content/doc/developer/plugin-governance/managing-permissions.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: developersection
+title: Managing plugin permissions
+references:
+- url: https://github.com/jenkins-infra/repository-permissions-updater
+  title: Repository Permissions Updater
+- url: ../adopt-a-plugin
+  title: Adopting a plugin
+---
+
+This page documents managing permissions for plugin maintainers.
+There are 2 main permissions available to plugin contributors:
+
+* **GitHub permissions**. 
+  It includes merging pull requests, push to the repositories, managing GitHub applications, etc.
+  The effective list of permissions depends on the link:https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization[permission levels], see below.
+* **Release Permissions** which are needed to deploy releases to the link:https://repo.jenkins-ci.org/webapp/#/home[Jenkins artifact repository].
```

Comment:
Do you need to mention this is how plugins are released to the update center as well?

Reply:
Probably better in https://jenkins.io/doc/developer/publishing/artifact-repository/